pkg/collector/corechecks/snmp/checkconfig/testing.go

```diff
@@ -0,0 +1,24 @@
+package checkconfig
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+// SetConfdPathAndCleanProfiles is used for testing only
+func SetConfdPathAndCleanProfiles() {
```

Comment:
Does this work when called from a file in a different path than this utils? Looking at the `Join("."`

Reply:
I used a workaround to have it work at 2 levels `.` and `..`:

```
	file, _ := filepath.Abs(filepath.Join(".", "test", "conf.d"))
	if !pathExists(file) {
		file, _ = filepath.Abs(filepath.Join("..", "test", "conf.d"))
	}
```

It's not that pretty, but didn't found another easy way around that. Let me know if you think there is a better way.